examples/streaming/microphone/main.py

```diff
@@ -30,31 +31,43 @@ def main():
         dg_connection = deepgram.listen.live.v("1")
 
         def on_open(self, open, **kwargs):
-            print(f"\n\n{open}\n\n")
+            print(f"Deepgram Connection Open")
 
         def on_message(self, result, **kwargs):
+            global is_finals
             sentence = result.channel.alternatives[0].transcript
             if len(sentence) == 0:
                 return
-            print(f"speaker: {sentence}")
+            if result.is_final:
+                is_finals.append(sentence)
+                if result.speech_final:
+                    utterance = ' '.join(is_finals)
+                    print(f"Speech Final: {utterance}")
+                    is_finals = []
+            else:
+                print(f"Interim Results: {sentence}")
 
         def on_metadata(self, metadata, **kwargs):
-            print(f"\n\n{metadata}\n\n")
+            print(f"Deepgram Metadata: {metadata}")
 
         def on_speech_started(self, speech_started, **kwargs):
-            print(f"\n\n{speech_started}\n\n")
+            print(f"Speech Started")
 
         def on_utterance_end(self, utterance_end, **kwargs):
-            print(f"\n\n{utterance_end}\n\n")
+            global is_finals
+            if len(is_finals) > 0:
+                utterance = ' '.join(is_finals)
+                print(f"Utterance End: {utterance}")
+                is_finals = []
 
         def on_close(self, close, **kwargs):
-            print(f"\n\n{close}\n\n")
+            print(f"Deepgram Connection Closed")
 
         def on_error(self, error, **kwargs):
-            print(f"\n\n{error}\n\n")
+            print(f"Deepgram Handled Error: {error}")
 
         def on_unhandled(self, unhandled, **kwargs):
-            print(f"\n\n{unhandled}\n\n")
+            print(f"Deepgram Unhandled Error: {unhandled}")
```

Comment:
It looks like if the PR is describing what these events are, the `on_handled` events arent errors, but rather events that aren't implemented in the SDK.